runtime/browser/xwalk_browser_main_parts.cc

```diff
@@ -217,6 +218,11 @@ void XWalkBrowserMainParts::PreMainMessageLoopRun() {
     xwalk_runner_->EnableRemoteDebugging(port);
   }
 
+#if !defined(OS_ANDROID)
```

Comment:
sad that we have "#if !defined(OS_ANDROID)" here. Could it be resolved in platform-specific media parts files?